README.md

```diff
@@ -350,9 +350,40 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
 
 The `patch` command patches the given resource.
 
+#### Patching account
+
+Use the `account` subcommand to patch an account by passing the ID. Pass the JSON body for the patch request in terminal using `echo '{<PATCH_BODY>}' | ` before the actual command. By default the dry run flag will be enabled. Pass `dryRun=false` flag to actually patch the resource.
+
+The following flags are available for `patch account`:
+
+```
+--dryRun                     If false, it will execute the patch command call in instead of a dry run.
+-h, --help                   help for patch
+```
+
+##### Examples
+
+* Patch an account by its ID and change the first name to 'Red Hat' (dry run) `echo '{ "first_name": "Red Hat" }' | ocm support patch account [accID]`
+
+#### Patching accounts
```

Comment:
In general, I think the patch accounts, organizations and subscriptions commands should appear before the patch account, organization, subscription - since they are more powerful and we would like to give them more visibility.

Reply:
Sorry, I think that I was not clear.

What I meant was to have the bulk action before the single resource action. Something like:

patch organizations
patch organization
patch subscriptions
patch subscription
patch accounts
patch account

Does it make sense?

Reply:
Yes it does. Changed the structure to have filter based commands before the singular patch commands